science/gmt5/Portfile

```diff
@@ -76,7 +76,9 @@ if {[variant_isset debug]} {
 
 configure.cflags-append     -fstrict-aliasing
 configure.args-append       -DDCW_ROOT=${prefix}/share/gmt/dcw \
+                            -DCOPY_DCW=off \
                             -DGSHHG_ROOT=${prefix}/share/gmt/gshhg \
+                            -DCOPY_GSHHG=off \
```

Comment:
These flags apply to all of the subports of this port (gmt5 and gmt6) but you've only changed the version/revision of one of those subports (gmt6). I assume that changing these flags changes the installed software. If that's so, then the revision of gmt5 should be increased so that this change will have the desired effect on it as well.

Reply:
The flags COPY_DCW and COPY_GSHHG are default to `off` in GMT 5 and GMT 6.0. Adding these two lines don't change the GMT5 port at all.

In GMT 6.1, the defaults are changed to `on`, but macports needs them to be `off`.

Reply:
Got it. Thanks.